### PR TITLE
Reward models

### PR DESCRIPTION
**Pull request by semioticrobotic**
This proposed chapter begins addresses #214 by offering an overview of various open source community reward models.

### CODE REVIEW

File: incentivizing_rewarding_participants.adoc
Comment (shaunix) [inline incentivizing_rewarding_participants.adoc:116]: "desire to a professional network" reads weird to me.

File: incentivizing_rewarding_participants.adoc
Comment (semioticrobotic) [reply incentivizing_rewarding_participants.adoc:59]: Plenty of available synonyms! I will choose another.

File: incentivizing_rewarding_participants.adoc
Comment (semioticrobotic) [reply incentivizing_rewarding_participants.adoc:78]: Since this might be a reminder that's fairly geographically specific, I think I can just remove it. Maybe too alarmist. Nice call.

File: incentivizing_rewarding_participants.adoc
Comment (semioticrobotic) [reply incentivizing_rewarding_participants.adoc:82]: This is a really great point, and I agree. Great instinct here. I will change.

**Comment by semioticrobotic** [comment]
Hey @lmaffeo! Any thoughts on this chapter to share? I'm eager to hear your impressions.

**Comment by semioticrobotic** [comment]
Hi @lmaffeo. Just wanted to check on this one more time. I'd like to finalize the chapter by the end of the calendar year, so eager to hear your feedback and suggestions.

If you'd prefer to discuss privately, of course that's fine! Just drop me a line.

**Comment by quaid** [comment]
> Hi @lmaffeo. Just wanted to check on this one more time. I'd like to finalize the chapter by the end of the calendar year, so eager to hear your feedback and suggestions.
> 
> If you'd prefer to discuss privately, of course that's fine! Just drop me a line.

Last I heard from Lauren several weeks ago, she was going into a heads-down mode for work for the rest of the year, and wasn't sure she'd be able to do a review in time.

Instead, I can seek out another reviewer, @Nebrethar comes to mind as an SME. I'm also happy to fill-in but my subject matter expertise is thinner.

**Comment by lmaffeo** [comment]
Hey guys, @quaid is correct: I emailed him at the start of November offering to do a SME review this winter (Jan/Feb, post-holidays) if it's possible to do at that time. I wasn't able to review this the last weekend of October as planned, so I can't put more on my own plate for the rest of the year. Let me know if a revised timeline works for you both. If not, feel free to choose another reviewer. Thanks, and happy Thanksgiving!

File: incentivizing_rewarding_participants.adoc
Comment (Nebrethar) [inline incentivizing_rewarding_participants.adoc:53]: The point system is a good fit with leaderboards, but they need to be further distinguished i.e.

Point systems on their own
Leaderboards on their own
Similarities and differences between the two.

File: incentivizing_rewarding_participants.adoc
Comment (Nebrethar) [inline incentivizing_rewarding_participants.adoc:44]: The "gaming the system" aspect of incentivization is a really important part of incentivizing successfully. 

I've seen it discussed often in software badging, where the rewards may be automated and relatively easy to obtain. If you separate badging from point systems/leaderboards, this would be a great topic of discussion.

File: incentivizing_rewarding_participants.adoc
Comment (Nebrethar) [inline incentivizing_rewarding_participants.adoc:49]: I'd like to argue that point systems and leaderboards should be a separate section from badging altogether. It is just so different from the former two.

Badging may be applied to projects, events, individuals, organizations, etc. and it has completely different roots.

File: incentivizing_rewarding_participants.adoc
Comment (Nebrethar) [inline incentivizing_rewarding_participants.adoc:127]: I don't completely disagree, but I would like to see more support for this statement.

File: incentivizing_rewarding_participants.adoc
Comment (Nebrethar) [reply incentivizing_rewarding_participants.adoc:44]: Here's an issue where some of us with [IEEE SA Open](https://opensource.ieee.org/community-advisory-group/community-badging/about/-/issues/26) are working on strategies to mitigate gaming.

**Review by Nebrethar — COMMENTED** [review]
This is awesome! You are doing great work.

Thanks for bringing me on to review.

File: incentivizing_rewarding_participants.adoc
Comment (semioticrobotic) [reply incentivizing_rewarding_participants.adoc:81]: Nice catch. Thanks!

File: incentivizing_rewarding_participants.adoc
Comment (semioticrobotic) [reply incentivizing_rewarding_participants.adoc:70]: "You" are an open source community architect. This is the target audience for _The Open Source Way_ and the reader persona each chapter presupposes.

File: incentivizing_rewarding_participants.adoc
Comment (semioticrobotic) [reply incentivizing_rewarding_participants.adoc:95]: I'm more than happy to do this and will get on it. In the meantime, @Nebrethar, do you have any resources/citations to share?

File: incentivizing_rewarding_participants.adoc
Comment (semioticrobotic) [reply incentivizing_rewarding_participants.adoc:97]: This is a great catch. Thank you. I will add this (using your suggested prose as seed!).

File: incentivizing_rewarding_participants.adoc
Comment (semioticrobotic) [reply incentivizing_rewarding_participants.adoc:44]: Thanks for this reference! I will review and attempt to reference it.

File: incentivizing_rewarding_participants.adoc
Comment (semioticrobotic) [reply incentivizing_rewarding_participants.adoc:49]: This is a very good point and it dovetails nicely with the resource you shared in the previous comment. So I will do my best to extricate.

File: incentivizing_rewarding_participants.adoc
Comment (semioticrobotic) [reply incentivizing_rewarding_participants.adoc:127]: No sweat. I will augment this sentence to emphasize the connection between a diversity of reward models, participant motivations, and a multiplicity of incentives.

File: incentivizing_rewarding_participants.adoc
Comment (semioticrobotic) [reply incentivizing_rewarding_participants.adoc:53]: Well put and easy to do. Thanks!

**Comment by semioticrobotic** [comment]
Thanks so much, @Nebrethar, for your thoughtful comments and suggestions. I've taken then on board and revised the chapter accordingly. Just pushed that [revision to this branch](https://github.com/theopensourceway/guidebook/pull/219/commits/d1abd478c36914bebdd72698620438fd217742f9). I do not know if you have time for (or interest in) another glance, but you are certainly more than welcome to take one. Otherwise, I'll defer to @quaid for next steps.

File: incentivizing_rewarding_participants.adoc
Comment (Nebrethar) [inline incentivizing_rewarding_participants.adoc:83]: Thank you for including me! I'm going by Matt Cantu Snell now 😁 that's my only remaining request

File: incentivizing_rewarding_participants.adoc
Comment (Nebrethar) [inline incentivizing_rewarding_participants.adoc:85]: Suggestions:
Snell -> Cantu Snell
commnity -> community

Thanks!

**Review by Nebrethar — CHANGES_REQUESTED** [review]
Awesome work! Only a tiny change request now

**Comment by Nebrethar** [comment]
I'm glad to help. I just looked over the revisions - great work!

I've added a nitpick about my name and I'm really happy with the changes.

**Comment by semioticrobotic** [comment]
Yikes! Very sorry to mis-identify you in the draft, Matt. I will fix that right away.

**Comment by semioticrobotic** [comment]
And done. Thanks so much, @Nebrethar!

**Comment by semioticrobotic** [comment]
This has undergone copy edit and SME review and I believe is now ready for final review, @quaid.

**Comment by quaid** [comment]
> This has undergone copy edit and SME review and I believe is now ready for final review, @quaid.

Perfect, thanks, I'm on it.

Appreciate the prodding, I'm still grasping how to handle my GitHub notifications effectively and they are increasing all the time!

File: incentivizing_rewarding_participants.adoc
Comment (quaid) [inline incentivizing_rewarding_participants.adoc:22]: s/contributres/contributes/

File: incentivizing_rewarding_participants.adoc
Comment (quaid) [inline incentivizing_rewarding_participants.adoc:32]: There could be a paragraph break about here, with the goal of making it easier to read. Does that seem doable here in the middle?

File: incentivizing_rewarding_participants.adoc
Comment (quaid) [inline incentivizing_rewarding_participants.adoc:108]: The phrase "in their projects' is a bit confusing, as the subject are people not yet in the project looking to see themselves there. I think the solution is to make it "the project"? Or I'm missing the subject:object connection properly.

typofix:  s/joinging/joining/

File: incentivizing_rewarding_participants.adoc
Comment (quaid) [inline incentivizing_rewarding_participants.adoc:116]: I think the *emphasis* here isn't needed, just being mindful of not overusing emphasis across the chapter.

File: incentivizing_rewarding_participants.adoc
Comment (quaid) [inline incentivizing_rewarding_participants.adoc:147]: maintaners => maintainers

**Review by quaid — CHANGES_REQUESTED** [review]
Content looks really good, it's come together quite well. I had just a few typos to fix and felt safer leaving them in comments rather than trying to modify the PR. :)

**Comment by semioticrobotic** [comment]
Thanks for the extra catches, @quaid. I will fix them right quick.

File: incentivizing_rewarding_participants.adoc
Comment (semioticrobotic) [reply incentivizing_rewarding_participants.adoc:108]: Great catch. It was an errant word. Fixed!

File: incentivizing_rewarding_participants.adoc
Comment (semioticrobotic) [reply incentivizing_rewarding_participants.adoc:116]: Agree. Thanks.

**Comment by semioticrobotic** [comment]
I think I've got them all now. Ponged back to you to confirm. Thanks!

File: incentivizing_rewarding_participants.adoc
Comment (quaid) [inline incentivizing_rewarding_participants.adoc:109]: Oh no we both missed a common one!

joinging => joining

:)

**Review by quaid — CHANGES_REQUESTED** [review]
Looks good, I just noticed a typo we both missed before. :rofl: 

From here we need to find someone who'd like to be the managing editor to wrangle the current and desired parts of the 2.1 release ...

**Comment by semioticrobotic** [comment]
Great! Nice catch. Thanks. I will fix.

As for the managing editor: What does that mean and/or involve? Is that role documented somewhere we can review and actively socialize (on mailing lists and the like)?

**Review by quaid — APPROVED** [review]
Looks great! This makes the first 2.1 chapter complete.

**Comment by quaid** [comment]
Note about "complete" and the publishing schedule:  There will certainly be time for @lmaffeo to come in with a second SME review before the milestone is complete, which would be very appreciated for this important topic.

**Comment by semioticrobotic** [comment]
Great news! Thanks, @quaid. And I'm very glad to hear @lmaffeo may have an opportunity to review and comment after all. Cheers! :clinking_glasses:

**Pull request merged**
